<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllUsers</class>
 <widget class="QDialog" name="AllUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>All Users</string>
  </property>
  <widget class="QPushButton" name="push_back">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);
color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>461</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46, 52, 54);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);
color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);
color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>View Selected User Images</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>461</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 215, 207);</string>
   </property>
   <property name="text">
    <string>All Users</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected User is Offline</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
